app/src/main/java/org/mozilla/fenix/settings/LoginsFragment.kt

```diff
@@ -264,6 +287,14 @@ class LoginsFragment : PreferenceFragmentCompat(), AccountObserver {
         }
     }
 
+    private fun navigateToSaveLoginSettingFragment() {
+        if (findNavController().currentDestination?.id == R.id.loginsFragment) {
```

Comment:
Why does this condition on the current destination?  What happens if it doesn't?

Reply:
It used to be a race condition that if you pressed on two settings items at once time it would crash but we've fixed that in https://github.com/mozilla-mobile/fenix/pull/6999 so removing this!